Code/Tools/TestImpactFramework/Runtime/Code/Include/TestImpactFramework/TestImpactUtils.h

```diff
@@ -103,4 +103,42 @@ namespace TestImpact
 
     //! User-friendly names for the client test result types.
     AZStd::string ClientTestResultAsString(Client::TestResult result);
+
+    SuiteType SuiteTypeFromString(const AZStd::string& suiteType);
```

Comment:
question: Should this also have an API comment (it's just that all the others do 😝)

Reply:
fixed 👍